app/templates/main/top-search.html

```diff
@@ -0,0 +1,23 @@
+<div class="search__container govuk-grid-column-full">
+    <div class="search__container__content">
+        <p class="govuk-body search__heading">Search for digital records</p>
+        <form method="post" action="{{ url_for('main.search') }}">
+            {{ form.csrf_token }}
```

Comment:
this will need to change to method="get" and remove the csrf_token like the changes Colin has made on AYR-664

Reply:
replaced method to get on search, removed csrf token